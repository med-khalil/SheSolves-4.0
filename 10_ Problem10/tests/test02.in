5
20
psfjakccxikwsgqfbcqx
20
moavomzoingrjephuulo
20
lracyoivlzsbcwavjwfy
20
lnhlnrwmwcmcmpwqjpds
20
ugirswrjhxinrgqrisfp