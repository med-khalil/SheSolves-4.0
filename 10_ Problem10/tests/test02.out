7
5
6
8
7
